proposal/comp110_architecture.bib

```diff
@@ -1,9 +1,44 @@
-@article{shannon, 
-	author={Shannon, Claude E.}, 
-	journal={The Bell System Technical Journal}, 
-	title={A mathematical theory of communication}, 
-	year={1948}, 
-	volume={27}, 
-	number={3}, 
-	pages={379-423}
+@article{guo,
+	author={Kaiqiang Guo},
+	journal={Intelligent Information Technology and Security Informatics},
+	title={An Improved Algorithm of Collision Detection in 2D Grapple Games },
+	year={2010},
+	pages={328 - 331}
 }
+
+@article{Tan,
+	author={Tan, Yunlan},
+	journal={International Conference on Information Engineering and Computer Science, 2009. ICIECS 2009 },
+	title={The Design of Collision Detection Algorithm in 2D Grapple Games},
+	year={2009},
+	pages={1-4}
+}
+
+@article{Mulley,
+	author={Mulley, G},
+	journal={8th EUROSIM Congress on Modelling and Simulation},
+	title={The Construction of a Predictive Collision 2D Game Engine},
+	year={2013},
+	pages={68 - 72}
+}
+
+
+@article{Nykl,
+	author={Nykl, S.},
+	journal={IEEE Transactions on Visualization and Computer Graphics},
+	title={Interactive Mesostructures with Volumetric Collisions},
+	year={2014},
+	volume={20},
+	issue={7},
+	pages={970 - 982}
+}
+
+
+@article{Chen,
+	author={Chen, Jin-Liang; Liu, J.S; Wan-Chi Lee },
```

Comment:
Another supporting technique. Interesting to write about, but would not focus on this for paper.